mongo-oplog/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// TypeScript Version: 2.1
+// Type definitions for mongo-oplog v2.0.2
+// Project: https://github.com/cayasso/mongo-oplog
+// Definitions by: FinalDes <https://github.com/FinalDes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="eventemitter3" />
+
+export function MongoOplog(url: string, filter: any): OplogTrigger;
+
+export interface OplogTrigger {
+    tail(): any; // return promise
+    on(type: string, callback: (doc: any) => void): any; // return eventemitter3
```

Comment:
Works for me with that version. Make sure you're running just `tsc` (in the `mongo-oplog` directory) and not `tsc mongo-oplog-tests.ts`. If you're still getting an error, what is it?